modules/h12mediaBidAdapter.js

```diff
@@ -206,4 +217,32 @@ function getLocalDateFormatted() {
   return `${d.getFullYear()}-${two(d.getMonth() + 1)}-${two(d.getDate())} ${two(d.getHours())}:${two(d.getMinutes())}:${two(d.getSeconds())}`;
 }
 
+function inIframe () {
```

Comment:
why didn't you utilize `utils.inIframe()` ?

Reply:
Fixed, now `utils.inIframe()` used